docs/how-to-guides/sync-dicom-metadata-to-fhir.md

```diff
@@ -1,12 +1,90 @@
+1. [Deploy a FHIR Server](https://github.com/microsoft/fhir-server)
+1. [Deploy DICOM Cast](../quickstarts/deploy-dicom-cast.md)
+
+## Configure Authentication using Managed Identity
+
+Currently there are three types of authentication supported for both Azure API for FHIR and Medical Imaging Server for DICOM: Managed Identity, OAuth2 Client Credential and OAuth2 User Password. The authentication can be configured via the application settings by the appropriate values in the `Authentication` property of the given server. For details on the three types, see [DICOM Cast authentication](/converter/dicom-cast/docs/authentication.md).
+
+This section will provide an end to end guide for configuring authentication with Managed Identity.
+
+### Create a resource application for FHIR and DICOM servers
+
+For both your FHIR and DICOM servers, you will create a resource application in Azure. Follow the instructions below for each server, once for your Medical Imaging Server for DICOM and once for your FHIR Server.
+
+1. Sign into the [Azure Portal](https://ms.portal.azure.com/). Search for **App Services** and select the FHIR or DICOM App Service. Copy the **URL** of the App Service.
+1. Select **Azure Active Directory** > **App Registrations** > **New registration**:
+    1. Enter a **Name** for your app registration.
+    2. In **Redirect URI**, select **Web** and enter the **URL** of your App Service.
+    3. Select **Register**.
+1. Select **Expose an API** > **Set**. You can specify a URI as the **URL** of your app service or use the generated App ID URI. Select **Save**.
+1. Select **Add a Scope**:
+    1. In **Scope name**, enter *user.impersonation*.
+    1. In the text boxes, add an admin consent display name and admin consent description you want users to see on the consent page. For example, *access my app*.
+
+### Set the Authentication for your FHIR & DICOM App Services
+
+For both your FHIR and DICOM servers, you will set the Audience and Authority for Authentication. Follow the instructions below for each server, once for your Medical Imaging Server for DICOM and once for your FHIR Server.
+
+1. Navigate to the App Service that you deployed to Azure.
+1. Select **Configuration** to update the **Audience** and **Authority**:
+    1. Set the **Application ID URI** from the App Service as the **Audience**.
+    1. **Authority** is whichever tenant your application exists in, for example: ```https://login.microsoftonline.com/<tenant-name>.onmicrosoft.com```.
+
+### Update Key Vault for DICOM Cast
+
+1. Navigate to the DICOM Cast Key Vault that was created when you deployed DICOM Cast.
+1. Select **Access Policies** in the menu bar and click **Add Access Policy**.
+    1. Under **Configure from template**, select **Secret Management**.
+    1. Under **Select principal**, click **None selected**. Search for your Service Principle, click **Select** and then **Add**. 

```

Comment:
Nit: you're not looking for a "Service Principle" here. You're updating keyvault to allow yourself to manage the secrets. We could potentially just link to: https://docs.microsoft.com/en-us/azure/key-vault/general/assign-access-policy-portal